Warmup-1/makes10/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export function makes10(num1: number, num2: number): boolean {
+  return num1 + num2 == 10 || num1 == 10 || num2 == 10 ? true : false;
```

Comment:
The ternary operator is redundant in this case

Reply:
Noted. Thanks.